src/CodeGenerator/Program.cs

```diff
@@ -360,52 +411,65 @@ static void Main(string[] args)
-                        }
-                        if (exportedName.Contains("~")) { continue; }
-                        if (overload.Parameters.Any(tr => tr.Type.Contains('('))) { continue; } // TODO: Parse function pointer parameters.
+                        if (TypeInfo.SkippedFunctions.Contains(fd.Name)) { continue; }
 
-                        bool hasVaList = false;
-                        for (int i = 0; i < overload.Parameters.Length; i++)
+                        foreach (OverloadDefinition overload in fd.Overloads.Where(o => !o.IsMemberFunction && o.IsInternal == isInternal))
                         {
-                            TypeReference p = overload.Parameters[i];
-                            string paramType = GetTypeString(p.Type, p.IsFunctionPointer);
-                            if (p.Name == "...") { continue; }
+                            string exportedName = overload.ExportedName;
+                            if (exportedName.StartsWith("ig"))
+                            {
+                                exportedName = exportedName.Substring(2, exportedName.Length - 2);
+                            }
+                            if (exportedName.Contains("~")) { continue; }
+                            if (overload.Parameters.Any(tr => tr.Type.Contains('('))) { continue; } // TODO: Parse function pointer parameters.
 
-                            if (paramType == "va_list")
+                            bool hasVaList = false;
+                            for (int i = 0; i < overload.Parameters.Length; i++)
                             {
-                                hasVaList = true;
-                                break;
+                                TypeReference p = overload.Parameters[i];
+                                string paramType = GetTypeString(p.Type, p.IsFunctionPointer);
+                                if (p.Name == "...") { continue; }
+
+                                if (paramType == "va_list")
+                                {
+                                    hasVaList = true;
+                                    break;
+                                }
                             }
-                        }
-                        if (hasVaList) { continue; }
+                            if (hasVaList) { continue; }
 
-                        KeyValuePair<string, string>[] orderedDefaults = overload.DefaultValues.OrderByDescending(
-                            kvp => GetIndex(overload.Parameters, kvp.Key)).ToArray();
+                            KeyValuePair<string, string>[] orderedDefaults = overload.DefaultValues.OrderByDescending(
+                                kvp => GetIndex(overload.Parameters, kvp.Key)).ToArray();
 
-                        for (int i = overload.DefaultValues.Count; i >= 0; i--)
-                        {
-                            if (overload.IsMemberFunction) { continue; }
```

Comment:
Here is the line I removed and added to the main loop to filter out member functions.